tasks/src/main/java/taf/product/pastebin/constants/Constants.java

```diff
@@ -0,0 +1,23 @@
+package taf.product.pastebin.constants;
+
+public class Constants {
```

Comment:
To common name. Please combine with the StringUtils class or give a more meaningful name